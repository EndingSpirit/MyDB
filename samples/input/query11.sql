SELECT Sailors.B, Sailors.C FROM Sailors, Reserves WHERE Sailors.A = Reserves.G GROUP BY Sailors.B, Sailors.C ORDER BY Sailors.C, Sailors.B;
