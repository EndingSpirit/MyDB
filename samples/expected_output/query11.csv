100, 50
200, 50
100, 105
200, 200
